test_data/rules/unused_top_members_in_executable_libraries.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2022, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// test w/ `dart test -N unused_top_members_in_executable_libraries`
+
```

Comment:
Could you add tests for both a named an an unnamed extension?

Reply:
Er, I guess not unnamed, because that is handled by the `unused_element` hint.

Reply:
Named extension is now covered by https://github.com/dart-lang/linter/pull/3513/files#diff-4f5160f7cce759c5604eb650859c439c3084aee4efd0b42cdd07d2bb8d3f69f1R109